libbeat/reader/syslog/rfc3164_test.go

```diff
@@ -99,10 +110,6 @@ var parseRFC3164Cases = map[string]struct {
 		In:      "<-1>Oct 11 22:14:15 test-host this is the message",
 		WantErr: ErrPriority,
 	},
-	"err-pri-missing-brackets": {
```

Comment:
Why did you remove this test case? It is true that priority might be missing, but what if the priority is supposed to be there but formatted incorrectly? I think this test case is still valid.

Reply:
It isn't, at least in terms of checking if the priority is correct or not. The parser looks for a starting `<`, if it's not present, then it goes to the next state, which is the timestamp. It can do this now since priority values are now optional. Since we start with a number here, it will try to parse it as an RFC3339 timestamp, which will ultimately fail.

I'll restore the test, but it will have to switch to checking for an `ErrTimestamp` instead of `ErrPriority` for the reason stated above.